data/campaigns/Eastern_Invasion/scenarios/08_Xenophobia.cfg

```diff
@@ -125,6 +125,7 @@
         [/modifications]
         controller=ai
         team_name=orc
+        #po: Whitefang Orcs, the same clan that attacked Tath at the start of Descent into Darkness
```

Comment:
Why does this specifically need a po?

Reply:
Translators can refer to their language's translation of DiD and use exactly the same name.

It was previously a non-po comment (somewhere in this file), and as it was useful for translation I promoted it to a po copy in two places in the file.